tools/ndt/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+set(NBL_EXTRA_SOURCES
+	# TODO: Cypi list extra sources if you need
+)
+
+nbl_create_executable_project("${NBL_EXTRA_SOURCES}" "" "" "" "${NBL_EXECUTABLE_PROJECT_CREATION_PCH_TARGET}")
```

Comment:
@AnastaZIuk take a look at this CMake and fix it

Reply:
maybe it shouldn't use the PCH target, cause its not a huge win here (NSC doesn't use anything from `examples_tests/common`)